ext/gd/tests/imagescale.phpt

```diff
@@ -0,0 +1,55 @@
+--TEST--
```

Comment:
Hmm, we didn't have an `imagescale()` by now? That should be fixed as soon as possible; I suggest that you rename this test to `imagescale_preserve_ratio.phpt` or something like that.